src/include/stir/DiscretisedDensity.inl

```diff
@@ -304,4 +304,29 @@ get_indices_closest_to_LPS_coordinates(const CartesianCoordinate3D<float>& coord
   return round(this->get_index_coordinates_for_LPS_coordinates(coords));
 }
 
+template<int num_dimensions, typename elemT>
+CartesianCoordinate3D<float>
+DiscretisedDensity<num_dimensions, elemT>::
+get_relative_coordinates_for_gantry_coordinates
+(const CartesianCoordinate3D<float>& coords,
+ const shared_ptr<ProjDataInfo> proj_data_info_ptr) const
+{
+  // gantry coordinates are like relative coordinates, but the (0, 0, 0)
```

Comment:
very confusing comment. "like"? and bed position should play here.

Anyway, I don't think we should have this function (See other comment)